build.roc

```diff
@@ -42,8 +42,6 @@ run = \{ debugMode, maybeRoc } ->
 
     rocVersion! rocCmd
 
-    generateGlue! rocCmd
-
```

Comment:
I don't completely understand why this is being removed now, can you explain?

Reply:
Yes, the "glue" only provide roc_std, and I think using cargo to get this directly from roc-lang/roc is a much better method for now. In future we may introduce glue back into the platform, but we have replaced all the interface types with hand-rolled implementations to be sure we don't have bugs lurking from RustGlue's current implementation.